api/etcdserverpb/raft_internal.proto

```diff
@@ -61,9 +66,9 @@ message InternalRaftRequest {
   AuthRoleGrantPermissionRequest auth_role_grant_permission = 1203;
   AuthRoleRevokePermissionRequest auth_role_revoke_permission = 1204;
 
-  membershippb.ClusterVersionSetRequest cluster_version_set = 1300;
-  membershippb.ClusterMemberAttrSetRequest cluster_member_attr_set = 1301;
-  membershippb.DowngradeInfoSetRequest  downgrade_info_set = 1302;
+  membershippb.ClusterVersionSetRequest cluster_version_set = 1300 [(versionpb.etcd_version_field) = "3.5"];
```

Comment:
Just trying to understand, will these changes need to be backported to 3.5 release as well? Would this create any compatibility issues?

Reply:
No, storage downgrade is done by older binaries. It's enough to have those annotations in v3.6, so it can verify the WAL and mark it as v3.5 compatible. v3.5 doesn't need to now about annotations.